test/backend/routes.spec.js

```diff
@@ -312,6 +368,51 @@ describe('Routes - ', () => {
         })).catch(done);
     });
 
+    // Settings
+    it('settings/urls - returns 200 and the urls', function(done) {
+        GET('settings/urls')
+        .then(res => res.json().then(json => {
+            assert.equal(res.status, 200);
+            assert.equal(json.http, 'http://localhost:9494');
+            assert.equal(json.https, '');
+            done();
```

Comment:
could we add `assert.equal(R.keys(json), ['http', 'https'])`. we want to make sure that secret settings aren't accidently leaked

Reply:
or a `deepEqual` like the settings test below